packages/icons/src/types.ts

```diff
@@ -7,6 +7,7 @@ const icons = [
 	'analyze',
 	'animal-cat',
 	'animal-turtle',
+	'anthropic',
```

Comment:
@tiffanylphan can you update this list?

Reply:
Yeah I'll let you know when it's ready again, sorry I'm in between things

Reply:
No rush!